web-client/public/js/grnsight.js

```diff
@@ -5,6 +5,9 @@ const container = require("./container").container;
 const displayStatistics = require("./graph-statistics").displayStatistics; // eslint-disable-line no-unused-vars
 const upload = require("./upload").upload;
 const nodeColoringController = require("./node-coloring").nodeColoringController;
+import setupHandlers from "setuphandlers";
+import grnState from "grnstate";
 
+setupHandlers(grnState);
```

Comment:
These changes, as they stand, look good. I agree that this is how GRNsight should start up. You should also add `updateApp` here. The initialization sequence is: set up all event handlers, then make the GRNsight UI display the values of `grnState` for the first time. So:
```javascript
setupHandlers(grnState);
updateApp(grnState);
```